scripts/init.sh

```diff
@@ -20,16 +20,19 @@ prompt_for_env() {
 		return
 	fi
 
-	read -rp "Enter PostgreSQL User: " postgres_user
-	read -rsp "Enter PostgreSQL Password: " postgres_password
-	echo
-	read -rp "Enter PostgreSQL Database Name: " postgres_db
+	[ -z "${POSTGRES_USER}" ] && read -rp "Enter PostgreSQL User: " POSTGRES_USER
```

Comment:
What is the point of checking for the values unless you're sourced/exported the .env file? And right after early return if .env file is found? Do these values exist in the environment already somehow?
Oh, you told the user to source the .env file themselves. Isn't this a step back towards what the Makefile initially did? Why would you want the variables to be restrained to one shell session instead of having the .env fetched from a backup location like it used to?
```
read -p "Wish to export current env? y/n: " -n 1 -r
if [[ $REPLY =~ ^[Yy]$ ]]
then
    set -a; source ${ENV_FILE_GLOBAL}; set +a
fi
echo
```
(set -a works in bash as well)

Reply:
The aim is to allow the user to export the .env in its shell after a first make, so that there is no need to retype the values for each variable after a make re, or make ffclean. It was indeed very painful to have to retype the values of the API keys after rebuilding the project.

Reply:
Then why do you just tell the user to do it if you know it's a pain to retype the values? Why not do it by default?

Reply:
That's what I tried to do first but did not find any way to export these variables on the user's shell. It appears it is only possible to export them in the environment of the script itself, until its termination. However, if you know a way to do that, I'll be happy to read your advice.

Reply:
Good point, I had forgotten that the variables were isolated like that. Tricky stuff. I don't understand why you'd delete the .env file to begin with, you could just change the value of SERV_IP with an easy `sed -i` command, that would allow you to prompt the user for the values of each variables only once, when the .env file is first created?